### PR TITLE
QAM: Fix srv_sync_all_products call in yaml

### PR DESCRIPTION
**Pull request by srbarrios**
## What does this PR change?

Fix a small typo after renaming a feature file.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed

- [x] **DONE**

## Test coverage
- Cucumber tests were changed

- [x] **DONE**

## Links

Ports:
- Manager-4.0: 
- Manager-4.1: 

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by ktsamis — APPROVED** [review]
LGTM

**Review by lkotek — APPROVED** [review]
LGTM :+1:

File: testsuite/run_sets/build_validation_add_custom_repositories.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # This yaml file contains all the features performing the creation of custom channels
 # for each Maintenance Update
 
-## Add MU repositories BEGIN ###
+## Add Custom repositories BEGIN ###
```
Comment (Bischoff) [inline testsuite/run_sets/build_validation_add_custom_repositories.yml:6]: ok, i have very sheldonish remark :-)

any reason for the upper case ? ;-)

**Review by Bischoff — APPROVED** [review]
see sheldonish nitpick

File: testsuite/run_sets/build_validation_add_custom_repositories.yml (hunk 1 above)
Comment (srbarrios) [reply testsuite/run_sets/build_validation_add_custom_repositories.yml:6]: rf sorry, I didn't saw it and merged already the other one.
Will fix in a future one.

Comment (Bischoff) [reply testsuite/run_sets/build_validation_add_custom_repositories.yml:6]: ok, no big deal @srbarrios

**Pull request merged**